### DISCUSSION TITLE
Is there a way to completely remove the background?

### DISCUSSION THREAD

**Discussion by gbraad** [c-smile/sciter-js-sdk#260] [Q&A] [answered]
I have made a very simple desktop clock that hovers over the other windows, however it has a blur.
Is it possible to completely remove the background and make it completely see-through?

![image](https://user-images.githubusercontent.com/1894/148632910-6be762c4-8326-4f88-80ac-08df23faa97e.png)

**Comment by pravic** [comment]
https://github.com/c-smile/sciter-js-sdk/blob/main/samples.sciter/window/chrome-types/window-frame-transparent.htm

**Comment by gbraad** [comment]
When the frame is transparent as suggested, is there an option to 'drag' the window to another location ?

Note: `window-blurBehind` has to be removed

**Comment by c-smile** [comment, marked as answer]
In the document above there is an element having `role="window-caption"` attribute: https://github.com/c-smile/sciter-js-sdk/blob/main/samples.sciter/window/chrome-types/window-frame-transparent.htm#L50

That literally gives the element role of window caption - area that can be used to drag the window by mouse.

**Reply by gbraad** [reply]
The frame has to be given the value of `transparent` and the element to allow dragging should have the `window-caption` role.

**Comment by gbraad** [comment]
Oh darn... I see... the SVG has to be given the `caption-role`. Let me check.

**Comment by gbraad** [comment]
That worked... but I still see that the frame at the top claims pixels that aren't expected.

![image](https://user-images.githubusercontent.com/1894/148651087-f2aeec37-7782-45f2-a93a-eb639d99af51.png)

What could that be ?

**Comment by gbraad** [comment]
My mistake,. Still had some CSS for the `header`

**Comment by c-smile** [comment]
How your root element of the window is defined?

Should be something like 
```HTML
<html 
  window-frame="transparent" 
  window-width=420px 
  window-height=420px >
  ```
where   `window-frame="transparent"` declares borderless window.

Also check that the border is not declared in your CSS.

**Comment by gbraad** [comment]
> Also check that the border is not declared in your CSS.

That is what happened as I was busy tweaking this. Resolved. Thanks!
